org_fedora_oscap/common.py

```diff
@@ -341,6 +342,47 @@ def run_oscap_remediate(profile, fpath, ds_id="", xccdf_id="", tailoring="",
     return proc.stdout
 
 
+def _schedule_firstboot_remediation(
+        chroot, profile, ds_path, results_path, report_path, ds_id, xccdf_id, tailoring_path):
+    config = textwrap.dedent(f"""\
+    OSCAP_REMEDIATE_DS='{ds_path}'
+    OSCAP_REMEDIATE_PROFILE_ID='{profile}'
+    OSCAP_REMEDIATE_ARF_RESULT='{results_path}'
+    OSCAP_REMEDIATE_HTML_REPORT='{report_path}'
+    OSCAP_REMEDIATE_VERBOSE_LOGS='/var/tmp/oscap_verbose.log'
```

Comment:
After the reboot, I can't find this file.

Also, I don't like that this is located or should be located in `/var/tmp` when other outputs are saved in `/root/openscap_data`. I would put all in a single directory.

Reply:
We traditionally don't save oscap logs, I put that there for debugging purposes, but I don't recall whether it worked. I will probably remove that line.

Reply:
@matejak So will you remove ir or won't you remove it?

Reply:
removed.

Reply:
Will you do the same thing also in #190?